.github/dependabot.yml

```
@@ -0,0 +1,57 @@
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    target-branch: "master"
+    commit-message:
+      # Prefix all commit messages with "chore: "
+      prefix: "chore"
```

Looking at this example looks like we should include the ": " as well https://github.com/dependabot/dependabot-core/issues/10627
```suggestion
      prefix: "chore: "
```